pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -71,24 +73,29 @@ def aimsgb(self):
 
     def cut(self, *args, **kwargs):
         return self.ase.cut(*args, **kwargs)
+
```

Comment:
I think elsewhere we do the doc addition immediately adjacent to the function definition, so for consistency let's just keep doing that instead of these newlines.